website/docs/r/is_lb_pool.html.markdown

```diff
@@ -15,9 +15,9 @@ Create, update, or delete a VPC load balancer pool.  For more information, about
 ### Sample to create a load balancer pool.
 
 ```terraform
-resource "ibm_is_lb_pool" "testacc_pool" {
-  name           = "test_pool"
-  lb             = "addfd-gg4r4-12345"
+resource "ibm_is_lb_pool" "example" {
+  name           = "example_pool"
```

Comment:
use hyphen instead of underscores. Underscores are not allowed accorgin to the regex for name